test/run-php-linter.sh

```diff
@@ -18,4 +18,11 @@ vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php htdocs || exit
 # Run PHPCS on all modules
 vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/ || exit $?;
 
+# Run PHPCS on src/ directory using a different ruleset
+# FIXME Currently the files in the src/ directory do not meet the standards of
+# PSR2. Until they are cleaned up we cannot include this test in our static
+# analysis without breaking Travis. This is not a priority issue but at some
+# point the issues should be remedied and the following line uncommented.
+vendor/bin/phpcs --standard=docs/SrcCS.xml --extensions=php/php,inc/php src/ || exit $?;
```

Comment:
the inc/php is only required for the php/libraries directory, in src they should all have the php extension.